keps/prod-readiness/sig-scheduling/2249.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2249
+alpha:
+  approver: "@wojtek-t"
```

Comment:
Re-assign to me, Wojtek is over-subscribed

Reply:
done.